### DISCUSSION TITLE
Working with latest RaspberryMatic?

### DISCUSSION THREAD

**Discussion by mado0803** [thkl/hap-homematic#696] [Q&A]
Hello community,

I read the discussions of past compatibility problems with the CCU3 / RaspberryMatic releases starting around Nov 2023. I saw as well the comment that the project is still alive, but time is the limiting factor for updating it. All good and I eagerly wait for the update (and donated money for keeping it alive).

My question is: Did anyone update RaspberryMatic lately and can confirm hap-homematic is working again as expected with e.g. 3.77.6.20240720? I am too lazy to try it out and roll-back, I rather ask here 😬

Best,
Marco

**Comment by mado0803** [comment]
Hello community, 
Nobody tried yet the latest release? I'd love to update my RaspberryMatic, but for sure want to keep my HomeKit integration.
Best,
Marco

**Reply by ZenitES** [reply]
Version 3.77.7.20240826 also runs on my Rasberrymatic without any problems.

**Comment by timwegenerDE** [comment]
I‘m on 3.77.7.20240826 without any problem.

**Reply by mado0803** [reply]
Hello @timwegenerDE,

I just updated to that version and can't reach hap-homematic anymore :-( Restarts of the add-on or the whole system didn't help. I will try some more but in the end can only downport back to 3.3.*** or whatever my version before was.

I am happy for you that it works.

Best,
Marco

**Reply by mado0803** [reply]
Rolling back. And I meant I was on version 3.73.9.20231130. Not 3.3...

**Comment by timwegenerDE** [comment]
Maybe try a clean install with a restore from your backup?

**Reply by timwegenerDE** [reply]
On a different SD card of course…

**Reply by mado0803** [reply]
Thanks for the idea! I will consider it or stay for now on 3.73.

**Reply by murksmax66** [reply]
Hi,
I'm using 3.79.6.20241031 (rpi4), working with no major problems. Just HAP ist not starting after reboot - I have to start the extension manualy ...

**Comment by mado0803** [comment]
Hello everyone,
I had to follow https://github.com/thkl/hap-homematic/issues/672#issuecomment-2007954663 for removing the timeout. Now it just worked when upgrading to 3.79.6.20241122.

Best,
Marco
